scripts/pce-fw-builder.sh

```diff
@@ -1,9 +1,10 @@
 #!/bin/bash -x
 
-echo $*
-
 cd /home/coreboot/coreboot
 
+legacy=$1
```

Comment:
Has variable legacy any use in this script?

Reply:
@miczyg1 it is not used now, but I would like to keep because I think there will be customizations soon.